packages/terra-navigation-side-menu/src/NavigationSideMenu.jsx

```diff
@@ -173,10 +177,10 @@ class NavigationSideMenu extends React.Component {
   }
 
   buildListContent(currentItem) {
-    if (currentItem && currentItem.childKeys && currentItem.childKeys.length) {
-      return <List className={cx(['menu-list'])}>{currentItem.childKeys.map(key => this.buildListItem(key))}</List>;
-    }
-    return null;
+    return (currentItem && currentItem.childKeys && currentItem.childKeys.length
```

Comment:
not a big deal, but to me, this logic is harder to understand when it is a ternary statement